<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmConfigIpcSearch</class>
 <widget class="QWidget" name="frmConfigIpcSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <widget class="QFrame" name="frameRight">
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>210</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="9" column="0" colspan="2">
       <widget class="QLabel" name="labCount">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>共搜索到 0 个设备</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cboxSearchLocalIP">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cboxSearchFilter">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QComboBox" name="cboxSearchClear">
        <item>
         <property name="text">
          <string>自动清空</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>设备累加</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labSearchUserPwd">
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="btnSearchAll">
          <property name="text">
           <string>广播搜索</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnSearchOne">
          <property name="text">
           <string>单播搜索</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="btnAddAll">
          <property name="text">
           <string>添加选中</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btnMediaOne">
          <property name="text">
           <string>获取当前</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btnMediaAll">
          <property name="text">
           <string>获取所有</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btnAddOne">
          <property name="text">
           <string>添加当前</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labSearchUserName">
        <property name="text">
         <string>用户</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtSearchUserName">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="labSearchClear">
        <property name="text">
         <string>策略</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labSearchDeviceIP">
        <property name="text">
         <string>单播</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cboxSearchDeviceIP">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="XLineEdit" name="txtSearchUserPwd">
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labSearchLocalIP">
        <property name="text">
         <string>网卡</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labSearchInterval">
        <property name="text">
         <string>间隔</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labSearchFilter">
        <property name="text">
         <string>过滤</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="cboxSearchInterval"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="labSearchTimeout">
        <property name="text">
         <string>超时</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="cboxSearchTimeout"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XLineEdit</class>
   <extends>QLineEdit</extends>
   <header>xlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableWidget</tabstop>
  <tabstop>txtSearchUserName</tabstop>
  <tabstop>txtSearchUserPwd</tabstop>
  <tabstop>cboxSearchLocalIP</tabstop>
  <tabstop>cboxSearchDeviceIP</tabstop>
  <tabstop>cboxSearchFilter</tabstop>
  <tabstop>cboxSearchInterval</tabstop>
  <tabstop>cboxSearchTimeout</tabstop>
  <tabstop>cboxSearchClear</tabstop>
  <tabstop>btnSearchAll</tabstop>
  <tabstop>btnSearchOne</tabstop>
  <tabstop>btnMediaAll</tabstop>
  <tabstop>btnMediaOne</tabstop>
  <tabstop>btnAddAll</tabstop>
  <tabstop>btnAddOne</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
